app/components/ChatBox.js

```
@@ -0,0 +1,77 @@
+  const handleKeyDown = (e) => {
+    if (e.key === 'Enter' && !e.shiftKey) {
+      e.preventDefault();
+      handleSendMessage();
+    }
+  };
+  
+  return (
+    <div className="border border-gray-200 rounded-lg overflow-hidden flex flex-col bg-white max-w-2xl mx-auto mt-5 shadow-sm h-[300px]">
+      <div className="flex-1 overflow-y-auto p-4 chat-messages">
+        {messages.map((message, index) => (
+          <div key={index} className="clear-both mb-2">
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)